packages/arb-bridge-eth/contracts/bridge/Outbox.sol

```diff
@@ -244,7 +241,11 @@ contract Outbox is IOutbox, Cloneable {
         return MerkleLib.calculateRoot(proof, path, keccak256(abi.encodePacked(item)));
     }
 
-    function outboxesLength() public view returns (uint256) {
-        return outboxes.length;
+    function outboxEntriesLength() external view returns (uint256) {
+        return outboxEntries.length;
+    }
+
+    function outboxEntryExists(uint256 outboxEntryIndex) external view returns (bool) {
+        return outboxEntries[outboxEntryIndex].root != bytes32(0);
```

Comment:
Q: is the assumption of this existence check sound? (or should outboxEntries just include an `exists` bool)

Reply:
it's really the equivalent usage on line 181 that matters (this one's just for client side convenience)